src/Logging/src/DynamicSerilogBase/SerilogConfigurationExtensions.cs

```diff
@@ -4,11 +4,26 @@
 
 using Serilog;
 using Serilog.Configuration;
+using Serilog.Core;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
 
 namespace Steeltoe.Extensions.Logging.DynamicSerilog
 {
     internal static class SerilogConfigurationExtensions
     {
         public static LoggerConfiguration SerilogConsole(this LoggerSinkConfiguration sinkConfiguration) => sinkConfiguration.Console();
+
+        internal static LoggerConfiguration AddConsoleIfNoSinksFound(this LoggerConfiguration loggerConfiguration)
+        {
+            var configuredSinksField = loggerConfiguration.GetType().GetField("_logEventSinks", BindingFlags.NonPublic | BindingFlags.Instance);
```

Comment:
SonarCloud is (rightfully) complaining about use of reflection here to access a non-public field, but I'm not sure that there is any other way to detect whether a sink has already been added or not

Reply:
This _may_ be excessive as having two ConsoleSinks does not appear to result in duplicate logging, so maybe this is all just excessive ?

Reply:
One of the reasons I remember adding this is people trying it would remove the configured(sample) sink and wonder why it wasn't working (myself included). But I wasn't sure about dual logging - that has been an issue as well. I was thinking may be check for at console sink and add it if not present - so if they add some remote sink then still be able to see something in console - unless someone wants its really not to be present.